### DISCUSSION TITLE
PHP Deprecated:  Optional parameter $force declared before required parameter

### DISCUSSION THREAD

**Discussion by rich-williamson** [livewire/livewire#7321] [1. Help]
I upgraded from 3.1 to 3.2 today and I'm receiving the following deprecation warning in a number of places such as running artisan commands.

PHP Deprecated:  Optional parameter $force declared before required parameter $testType is implicitly treated as a required parameter in /var/www/html/vendor/livewire/livewire/src/Features/SupportConsoleCommands/Commands/MakeCommand.php on line 111.

Do I need to upgrade anything else alongside Livewire?

**Comment by rich-williamson** [comment]
Arh ignore me I've seen the PR for the fix. https://github.com/livewire/livewire/pull/7319

**Reply by joshhanley** [reply]
@rich-williamson All good 😊 will we tag the fix in a couple of days, incase there are any other bugs in the new release.
